modules/sdk/project-templates/project-templates-form-field/src/test/java/com/liferay/project/templates/form/field/ProjectTemplatesFormFieldTest.java

```diff
@@ -519,6 +519,77 @@ public void testBuildTemplateFormField73() throws Exception {
 		}
 	}
 
+	@Test
+	public void testBuildTemplateFormField73WithReactFramework()
+		throws Exception {
+
+		String liferayVersion = "7.3.3";
+		String name = "foobar";
+		String jsFramework = "react";
+
+		File workspaceDir = buildWorkspace(temporaryFolder, liferayVersion);
+
+		File gradleProjectDir = buildTemplateWithGradle(
+			new File(workspaceDir, "modules"), "form-field", name,
+			"--liferay-version", liferayVersion, "--js-framework", jsFramework);
+
+		testContains(
+			gradleProjectDir, "bnd.bnd", "Provide-Capability:", "soy;",
+			"type:String=\"LiferayFormField\"");
+		testContains(
+			gradleProjectDir, "build.gradle",
+			"compileOnly group: \"com.liferay\", name: " +
+				"\"com.liferay.dynamic.data.mapping.api\"",
+			"compileOnly group: \"com.liferay\", name: " +
+				"\"com.liferay.frontend.js.loader.modules.extender.api\"",
+			"jsCompile group: \"com.liferay\", name: " +
+				"\"com.liferay.dynamic.data.mapping.form.field.type\"",
+			DEPENDENCY_PORTAL_KERNEL);
+		testContains(
+			gradleProjectDir,
+			"src/main/java/foobar/form/field/FoobarDDMFormFieldType.java",
+			"com.liferay.frontend.js.loader.modules.extender.npm.NPMResolver;",
+			"org.osgi.service.component.annotations.Reference;",
+			"ddm.form.field.type.description=foobar-description",
+			"ddm.form.field.type.display.order:Integer=13",
+			"ddm.form.field.type.group=customized",
+			"public String getModuleName()",
+			"public boolean isCustomDDMFormFieldType()",
+			"private NPMResolver _npmResolver;");
+		testContains(
+			gradleProjectDir,
+			"src/main/resources/META-INF/resources/foobar.es.js",
+			"import React from 'react';",
+			"import {FieldBase} from 'dynamic-data-mapping-form-field-type" +
+				"/FieldBase/ReactFieldBase.es';",
+			"import {useSyncValue} from " +
+				"'dynamic-data-mapping-form-field-type/hooks/useSyncValue.es';",
+			"const Foobar = ({name, onChange, predefinedValue, readOnly, " +
+				"value})",
+			"const Main = ({label, name, onChange, predefinedValue, " +
+				"readOnly, value, ...otherProps})",
+			"Main.displayName = 'Foobar';", "export default Main;");
+
+		testNotContains(
+			gradleProjectDir, "build.gradle", true, "^repositories \\{.*");
+		testNotContains(gradleProjectDir, "build.gradle", "version: \"[0-9].*");
+
+		if (isBuildProjects()) {
+			executeGradle(
+				workspaceDir, _gradleDistribution,
+				":modules:" + name + GRADLE_TASK_PATH_BUILD);
+
+			File gradleOutputDir = new File(gradleProjectDir, "build/libs");
+
+			Path gradleOutputPath = FileTestUtil.getFile(
+				gradleOutputDir.toPath(), OUTPUT_FILE_NAME_GLOB_REGEX, 1);
+
+			Assert.assertNotNull(gradleOutputPath);
+
+			Assert.assertTrue(Files.exists(gradleOutputPath));
+		}
```

Comment:
@marcosapmf, can you please clarify it to me?

Reply:
@natocesarrego, I put this because it is present in `testBuildTemplateFormField73`, but, apparently, it is not necessary once there is not a `build/libs` folder when we generate the project template, I will remove this in the next pull